### PR TITLE
Range selection

### PR DESCRIPTION
**Pull request by tcanabrava**
Use PingControlSlider for the range selection

### CODE REVIEW

**Comment by patrickelectric** [comment]
I'm getting some errors while moving the control, also it starts in a weird position. 
image

```
[12:49:50.230] default[Warning]: Ping360ControlPanel.qml(32) qrc:/Ping360ControlPanel.qml:32: Error: Cannot assign [undefined] to double                                                                             
[12:49:50.230] default[Warning]: PingSlider.qml(32) qrc:/PingSlider.qml:32:9: Unable to assign [undefined] to QString                                                                                                
[12:49:50.277] default[Warning]: Ping360ControlPanel.qml(32) qrc:/Ping360ControlPanel.qml:32: Error: Cannot assign [undefined] to double                                                                             
[12:49:50.278] default[Warning]: PingSlider.qml(32) qrc:/PingSlider.qml:32:9: Unable to assign [undefined] to QString                                                                                                
[12:49:50.293] default[Warning]: Ping360ControlPanel.qml(32) qrc:/Ping360ControlPanel.qml:32: Error: Cannot assign [undefined] to double                                                                             
[12:49:50.293] default[Warning]: PingSlider.qml(32) qrc:/PingSlider.qml:32:9: Unable to assign [undefined] to QString                                                                                                
[12:49:50.299] default[Warning]: Ping360ControlPanel.qml(32) qrc:/Ping360ControlPanel.qml:32: Error: Cannot assign [undefined] to double                                                                             
[12:49:50.299] default[Warning]: PingSlider.qml(32) qrc:/PingSlider.qml:32:9: Unable to assign [undefined] to QString                                                                                                
[12:49:50.303] default[Warning]: Ping360ControlPanel.qml(32) qrc:/Ping360ControlPanel.qml:32: Error: Cannot assign [undefined] to double                                                                             
[12:49:50.304] default[Warning]: PingSlider.qml(32) qrc:/PingSlider.qml:32:9: Unable to assign [undefined] to QString                                                                                                
[12:49:50.323] default[Warning]: Ping360ControlPanel.qml(32) qrc:/Ping360ControlPanel.qml:32: Error: Cannot assign [undefined] to double                                                                             
[12:49:50.323] default[Warning]: PingSlider.qml(32) qrc:/PingSlider.qml:32:9: Unable to assign [undefined] to QString
```

File: qml/Ping360ControlPanel.qml [hunk 1]
```diff
@@ -24,14 +24,12 @@ Item {
...
-            PingSlider {
+            PingComboSlider {
                 Layout.fillWidth: true
                 text: "Range (m)"
-                value: Math.round(ping.range)
-                control.stepSize: 1
-                control.from: 1
-                control.to: 100
-                control.onMoved: ping.range = control.value
+                startingIndex: ping.range
```
Comment (patrickelectric) [inline qml/Ping360ControlPanel.qml]: The range is in 'mm' how can this be an index ?

File: qml/Ping360ControlPanel.qml [hunk 2]
```diff
@@ -24,14 +24,12 @@ Item {
...
+                model: [1, 2, 5, 10, 20, 30, 40, 50, 60]
+                onOptionChanged : ping.range = value
```
Comment (patrickelectric) [inline qml/Ping360ControlPanel.qml]: avoid space after property assignment. 
```suggestion
                onOptionChanged: ping.range = value
```

**Comment by patrickelectric** [comment]
@tcanabrava 	please merge after the CI tests.

**Comment by tcanabrava** [comment]
Ok

Em qua, 31 de jul de 2019 às 20:56, Patrick José Pereira <
notifications@github.com> escreveu:

**Pull request merged**